task-impls/src/consensus.rs

```diff
@@ -851,7 +851,7 @@ where
                             }
 
                             // warn!("Inserting leaf into storage {:?}", leaf.commit());
-                            if *view % 10 == 0 && self.quorum_exchange.is_leader(view){
+                            if *view % 10 == 0 && self.quorum_exchange.is_leader(view) {
```

Comment:
Out of curiosity, why `% 10`?

Reply:
I actually had put this logging to check that we were committing unique transactions.  I just wanted to log relatively less so it wasn't crowding out everything else in the logs.  We probably don't need this logging.  I think thought the most updated run_view_refactor code didn't have this tho.

Reply:
I changed it to `% 50` for now- if you'd rather I delete it, just say the word!